axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -475,12 +465,25 @@ InvoiceSet.length > 0))"
 		       ]]></viewer>
 			</field>
 		</panel>
-
+       
+        <panel-stack name="otherDetailsPanel" sidebar="true" hideIf="partner == null">
+			<panel name="referenceDocumentsPanel" title="Reference documents" itemSpan="6" canCollapse="true" collapseIf="true">
+				<field name="internalReference"/>
+				<field name="externalReference"/>
+			</panel>
+			<panel name="datesPanel" title="Dates" readonlyIf="statusSelect == 3" canCollapse="true" collapseIf="true">
+				<field name="invoiceDate" onChange="action-invoice-record-due-date,action-invoice-record-estimated-payment-date" readonlyIf="ventilatedByUser || ventilatedDate"/>
+				<field name="dueDate" onChange="action-invoice-record-estimated-payment-date"/>
+				<field name="estimatedPaymentDate" readonly="true" if-module="axelor-cash-management"  if="__config__.app.isApp('cash-management')" />
```

Comment:
Title of 'estimatedPaymentDate' doesn't displayed properly on small screen, put colSpan="12"